src/logging/monitor.ts

```diff
@@ -1,14 +1,18 @@
 import { pipeline } from 'stream/promises';
+import { setTimeout } from 'timers/promises';
 
 import { spawnJournalctl, toJournalDate } from '../lib/journald';
 import log from '../lib/supervisor-console';
-import { setTimeout } from 'timers/promises';
 
-export type MonitorHook = (message: {
+type MonitorHookMessage = {
```

Comment:
You could probably extract `LogMessage` from log-backend into its own module and use that here and in the backends

Reply:
Added [src/logging/types.ts](https://github.com/balena-os/balena-supervisor/pull/2382/commits/ebfcdc74ff396fd06bc5887687d0cc66bd8d7ae0#diff-b7efb7d611599542fae83eb6acd5685ad5a3af21ae51edfb220e87053497158fR1) which defines `BaseLogMessage`, `SystemLogMessage`, and `LogMessage` which can be either system or container log messages. Then [redefined](https://github.com/balena-os/balena-supervisor/pull/2382/commits/ebfcdc74ff396fd06bc5887687d0cc66bd8d7ae0#diff-ba81f47a86a6a2d1ec1033fcc6d92c30d75d3557278a367721609771c55f4662R8-R9) `MonitorHook` and `SystemMonitorHook` to refer to the relevant `LogMessage` type from types.ts.